packages/fxa-content-server/tests/functional/settings/connected_services_oauth_clients.js

```diff
@@ -90,14 +90,14 @@ describe('connected services: oauth clients', () => {
     // disconnect
     await click(
       // the current lack of unique ids make the positional selector here necessary
-      `${selectors.SETTINGS.CONNECTED_SERVICES.HEADER} #service:nth-child(3) ${selectors.SETTINGS.CONNECTED_SERVICES.SIGN_OUT}`
+      `${selectors.SETTINGS.CONNECTED_SERVICES.HEADER} .my-1.settings-connected-service:nth-child(3) ${selectors.SETTINGS.CONNECTED_SERVICES.SIGN_OUT}`
```

Comment:
Q: Should we add `.my-1.settings-connected-service` to `../lib/selectors` (ie: `selectors.SETTINGS.CONNECTED_SERVICES.SOMETHING`)?

Reply:
Similarly, the other selectors seem to be to `data-testid`, could this change do the same instead of using a className?